xbmc/guilib/GUIFontManager.h

```diff
@@ -81,6 +102,8 @@ class GUIFontManager : public IMsgTargetCallback
                                         std::string& current,
                                         void* data);
 
+  std::vector<std::string> GetUserFontsFamilyNames();
```

Comment:
would be nice to have docs for this one someone not familiar with the codebase won't understand what this does from the signature

Reply:
i have deleted the doc for mistake thanks